src/scene/mesh-instance.js

```diff
@@ -13,6 +14,7 @@ import {
 
 var _tmpAabb = new BoundingBox();
 var _tempBoneAabb = new BoundingBox();
+var _tempSphere = new BoundingSphere(null, 0);
```

Comment:
No point in passing these parameters - I wouldn't bother specifying them.

Reply:
excelent point!